evaluations/openbrush_2_takahser.md

```diff
@@ -0,0 +1,26 @@
+# Evaluation
+
+- **Status:** In Progress
+- **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/openbrush.md
+- **Milestone:** 2
+- **Kusama Identity:** Address
+- **Previously successfully merged evaluation:** -
+
+| Number | Deliverable | Accepted | Link | Evaluation Notes |
+| ------ | ----------- | -------- | ---- |----------------- |
+| 0a. | License |<ul><li>[x] </li></ul>|[LICENSE](https://github.com/Supercolony-net/openbrush-contracts/blob/74524fb06a788aae011c3213561038a66ed7cac5/LICENSE)|MIT License|
+| 0b.  | Documentation |<ul><li>[x] </li></ul>|[Macros](https://github.com/Supercolony-net/openbrush-contracts/blob/783406805c4ef86176d4a3500db6840c5b7f059a/utils/brush/proc_macros/lib.rs), [Documentation](https://supercolony-net.github.io/openbrush-contracts/index.html)|<ul><li>all macros have extensive documentation</li><li>new features are documentated</li><li>the Rust code examples on the documentation fail to execute</li></ul>|
+| 0c. | Testing Guide |<ul><li>[] </li></ul>|[Macros](https://github.com/Supercolony-net/openbrush-contracts/blob/783406805c4ef86176d4a3500db6840c5b7f059a/utils/brush/proc_macros/lib.rs), [Examples](https://github.com/Supercolony-net/openbrush-contracts/tree/783406805c4ef86176d4a3500db6840c5b7f059a/examples), [Integration tests](https://github.com/Supercolony-net/openbrush-contracts/tree/783406805c4ef86176d4a3500db6840c5b7f059a/tests)|<ul><li>Macros include executable code examples as part of the comments</li><li>Various examples exists, e.g. for psp22, psp721 access control, ownable psp1155 tokens, reentrency guards, pausable contracts, timelocks, payment splitters</li><li>Integration tests exist. Since I was [unable to setup a node](https://github.com/paritytech/canvas-node/issues/75) I wasn't able to run them though.</li><li>Unit tests are failing on my machine.</li></ul>|
```

Comment:
Are they still failing? You mention a fix in the general notes.

Reply:
They're passing now, I forgot to update it.